Tests/test_imageops.py

```diff
@@ -404,6 +404,18 @@ def check(orientation_im):
     assert 0x0112 not in transposed_im.getexif()
 
 
+def test_exif_transpose_inplace():
+    with Image.open("Tests/images/orientation_rectangle.jpg") as im:
+        assert im.size == (2, 1)
+        assert im.getexif()[0x0112] == 8
```

Comment:
though hardcode `0x0112` is ok.
But why not change it to `ExifTags.Base.Orientation` according to src code?

Reply:
My thinking is that I know the [orientation tag is 0x0112](https://www.awaresystems.be/imaging/tiff/tifftags/orientation.html), and if Pillow's src code is changed so that the orientation tag is something different, I want this test to start failing.

Reply:
good